build.html

```diff
@@ -1,4 +1,4 @@
-<div class="passcode-wrapper" data-id="{{id}}" data-uuid="{{uuid}}">
+<div class="passcode-wrapper" data-passcode-id="{{id}}" data-passcode-uuid="{{uuid}}">
```

Comment:
I realised that calling it `data-lock-id` would make better sense, inline with the `com.fliplet.lock` name.

```suggestion
<div class="passcode-wrapper" data-lock-id="{{id}}" data-lock-uuid="{{uuid}}">
```

Everything else is good. I'll approve after this change.

Reply:
@tonytlwu Added this changes